engine/api/workflow/gorp_model.go

```diff
@@ -33,6 +33,9 @@ type NodeRun sdk.WorkflowNodeRun
 // JobRun is a gorp wrapper around sdk.WorkflowNodeJobRun
 type JobRun sdk.WorkflowNodeJobRun
 
+// JobRunInfo is a gorp wrapper around sdk.WorkflowNodeJobRunInfo
+type JobRunInfo sdk.WorkflowNodeJobRunInfo
```

Comment:
i think this is useless because you don't use to insert/select/update or delete in table workflow_node_run_job_info